packages/packages.go

```diff
@@ -44,3 +48,11 @@ func convertGopkgNameToPurl(name string) (rename string) {
 	}
 	return
 }
+
+func GimmeAPURL(name string, version string) string {
+	version = strings.Replace(version, "v", "", -1)
+	version = strings.Replace(version, "+incompatible", "", -1)
```

Comment:
Do we need to strip more here??  `+meta`, `-pre`, etc etc.

https://golang.org/ref/mod#versions

Reply:
I THINK I stripped that version part out for OSS Index, IIRC. I'd have to look further into it.